app/models/sock.rb

```diff
@@ -1,2 +1,6 @@
 class Sock < ApplicationRecord
+  has_many :sock_size
+
```

Comment:
Other half of has_many, through relationship

has_many :sizes, through: :sock_sizes